README.md

```diff
@@ -1,50 +1,51 @@
+This project contains an Unreal Editor plugin that works in conjunction with Visual Studio to display information about Blueprints assets in C++ code. 
 
-It can be installed in the Engine or Game project sources and is automatically invoked by VS when opening an Unreal Engine C++ project.
+The plugin can be installed in either the Engine or Game project sources, and it is automatically activated when an Unreal Engine project is opened in Visual Studio.
 
-## Prerequisites
+## Requirements
 
-1. Visual Studio 2022, with the "IDE features for Unreal Engine" component installed in the Gaming Workload.
+Before you begin, please make sure you have the following software and tools set up:
+
+1. Visual Studio 2022 has the "IDE features for Unreal Engine" component installed.
+   1. The component can be found in the "Game development with C++" workload or as an individual component.
 2. Unreal Engine, either installed or built from source.
-   1. See the following link for instructions on how to install or build the Engine: [Installing Unreal Engine](https://docs.unrealengine.com/5.0/en-US/installing-unreal-engine).
-   1. We have tested the source and instructions below using Unreal Engine versions 4.27, 5.0 and 5.1.
+   1. To learn how to install or build Unreal Engine, please refer to the following guide: [Installing Unreal Engine](https://docs.unrealengine.com/5.0/en-US/installing-unreal-engine).
+   1. The source code and instructions have been tested on Unreal Engine versions 4.27, 5.0 and 5.1.
 
-## Building and installing
+## Building and Installing the Plugin
 
-> Have an engine installed with the Epic Game Launcher and just want to use the plugin? You can skip the steps bellow and install from the Epic Marketplace using this link: <https://aka.ms/vsueplugin>.
+> If you have Unreal Engine installed and set up through the Epic Games Launcher, and you only want to use the plugin, you can skip the steps below and install it directly from the [Unreal Engine Marketplace](https://aka.ms/vsueplugin).
 
-The simplest way to use the plugin is to clone this repo under the `Plugins` folder of your game project or the engine source.
-If you have multiple projects in the same Visual Studio solution, it's recommended to use install the plugin at the engine level, and share the binaries across the projects.
+The most straightforward way to use the plugin is to clone the repo under the `Plugins` folder of your game project or engine source. If you have multiple projects in the same Visual Studio solution, it is recommended to install the plugin at the engine level and share the binaries across the projects.
 
-1. Clone the repo under the project plugin folder.
+1. Clone the repo under the project plugin folder by using the following commands:
 
    ```powershell
    cd <Project or Engine root folder>/Plugins
-   git clone <repo url>
+   git clone https://github.com/microsoft/vc-ue-extensions.git
```

Comment:
Strictly speaking, this is not portable across clones of the repository (e.g. when we moved from an internal repo to GH, we did not have to update this). I'm not against the change, but wanted to make sure we are clear on that part.

Reply:
This should be okay since I suspect more people won't need to change this